### ISSUE TITLE
Reimbursement for trade htrfa

### ISSUE THREAD

**Issue by polygonk** [bisq-network/support#1044]
Reimbursement for trade htrfa

I am BTC seller as maker
Trade amount and asset: 2 BTC for 2.73 ETH
Deposit amount: 0.3 BTC

Maker tx: 19f03a0f96cd99b727d5963b804fa6aa46fd95caa011967725a5456db418c7a5
Taker tx: 6f25ac0f5e2e9baed6797e7aaa8e43465e0409a20840b736bee0110777e0aeeb
Deposit tx:795ee63cd06e5b4b6ace35fb3e0b1c866e62441966e5944f5a4401e8f68f52fb
Delayed payout tx: 7506bc2cef3640900eb51fd4569d2a1ed034a9bb55f4b038061402ada2056cd4

**Reimbursement amount**: 1.2 BTC / 0.00003051 BSQ = 39,331.37 BSQ
Reason for reimbursement: seller accidentally created an offer below market price. Both parties came to an agreement.

#### Mediation suggestion

System message: Mediator's dispute summary:
Mediator's node address: 3z5jnirlccgxzoxc6zwkcgwj66bugvqplzf6z2iyd5oxifiaorhnanqd.onion:9999

Ticket closed on Nov 26, 2021 10:46:27 AM
Mediator node address: 7hkpotiyaukuzcfy6faihjaols5r2mkysz7bm3wrhhbpbphzz3zbwyqd.onion:9999

Summary:
Trade ID: htrfa
Currency: ETH
Trade amount: 2.00 BTC
Payout amount for BTC buyer: 0.38 BTC
Payout amount for BTC seller: 2.22 BTC

Reason for dispute: Other

Summary notes:
seller accidentally created an offer below market price. Both parties came to an agreement.

-----BEGIN SIGNATURE-----
302c021472dd3c08f05cef0c2ed7e9627b04deb6b2b50fdd02142ed4561b80e651c3a34f9ea6727dcee4cbde4c0b
-----END SIGNATURE-----

Next steps:
Open trade and accept or reject suggestion from mediator

#### Arbitration summary

Ticket closed on Dec 5, 2021 9:52:54 PM
Arbitrator node address: 3z5jnirlccgxzoxc6zwkcgwj66bugvqplzf6z2iyd5oxifiaorhnanqd.onion:9999

Summary:
Trade ID: htrfa
Currency: ETH
Trade amount: 2.00 BTC
Payout amount for BTC buyer: 0.30 BTC
Payout amount for BTC seller: 0.30 BTC

Reason for dispute: Other

Summary notes:
Seller accidentally created an offer below market price. Buyer took the offer and sent the ETH, but seller would not release the BTC as that would suppose a massive loss. Both traders agreed with the next payout with mediator and confirmed that later at arbitration:
1.1 BTC buyer
1.5 BTC seller
Mediator submitted a wrong mediation suggestion and buyer inadvertently opened arbitration.

Through this payout, arbitrator sends 0.3 BTC to each buyer and seller. As the amounts are too big to be fully paid in BTC, buyer should claim a reimbursement of 0.8 BTC to the DAO to be paid in BSQ, and seller should claim 1.2BTC in BSQ.

Arbitration secret: 61543

@refund-agent2 @leo816

**Comment by polygonk** [comment]
Yes, I see the 30 day average BSQ/BTC price as the same: 0.00003051.

That translates to a reimbursement of 39,331.37 BSQ

@refund-agent2

**Comment by refund-agent2** [comment]
1. Edit the rate and total reimbursement at the first post.
2. Send a DAO reimbursement request through DAO - Governance - Make a proposal - Reimbursement request and once submitted, copy the proposal txid.

**Comment by polygonk** [comment]
@refund-agent2 

I'm seeing this as the proposal txid:

http://mempool4t6mypeemozyterviq3i5de4kpoua65r3qkn5i3kknu5l2cad.onion/tx/a2f96800cb3c823e4552e239bd9b157ea644e4cdfead993c1450cfbf7884d26e

Is this correct ?

**Comment by refund-agent2** [comment]
Yes, it's correct.
Keep paying attention to this thread, but I think that there's nothing else to do on your side.

**Comment by refund-agent2** [comment]
The reimbursement proposal was accepted so @polygonk can exchange the XMR with @burningman3.
Matrix is not working to me right now so I can't coordinate the trade.

**Comment by leo816** [comment]
@refund-agent2 was this completed?

**Comment by refund-agent2** [comment]
I don't remember the details, but yes, I don't expect this user needing to reopen the issue after closed.
